src/utils/json/jsonParser.ts

```diff
@@ -74,6 +74,7 @@ export function parser(jsonStr: string): Graph {
       path: getNodePath(states.graph.nodes, states.graph.edges, node.id),
     }));
 
+    console.log(states.graph);
```

Comment:
It is good for best practice in removing logs

Reply:
Nice catch, it shouldn't be there!